### PR TITLE
Reown AppKit: version 1.0.0

### PR DESCRIPTION
**Pull request by tomiir**
# Description
Reown AppKit multichain version 1.0.0 release.

## Type of change

-  Chore (non-breaking change that addresses non-functional tasks, maintenance, or code quality improvements)

# Checklist

- [X] Code in this PR is covered by automated tests (Unit tests, E2E tests)
- [X] My changes generate no new warnings
- [X] I have reviewed my own code
- [X] I have filled out all required sections

### CODE REVIEW

File: .changeset/config.json [hunk 1]
```diff
@@ -1,10 +1,9 @@
 {
   "$schema": "https://unpkg.com/@changesets/config@3.0.0/schema.json",
-  "changelog": ["@changesets/changelog-github", { "repo": "WalletConnect/web3modal" }],
+  "changelog": ["@changesets/changelog-github", { "repo": "WalletConnect/shadow-appkit" }],
```
Comment (lukaisailovic) [inline .changeset/config.json:3]: Is this supposed to be shadow?

Comment (tomiir) [reply .changeset/config.json:3]: no, thank you

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: false
 contact_links:
   - name: ❓ Ask a question
-    url: https://github.com/WalletConnect/walletconnect-monorepo/discussions/new?category=web3modal
```
Comment (magiziz) [inline .github/ISSUE_TEMPLATE/config.yml:4]: No categories ?

File: apps/laboratory/src/components/AppKitInfo.tsx [hunk 3]
```diff
@@ -12,8 +12,8 @@ import {
 
 type AppKitInfoProps = {
   address?: string
-  chainId?: number
-  clientId: string | null
+  chainId?: number | string
+  clientId: string | undefined
```
Comment (magiziz) [inline apps/laboratory/src/components/AppKitInfo.tsx:16]:
```suggestion
  clientId?: string
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -11,7 +11,7 @@ body:
         You should also test your issue against the latest NPM package version(s) to ensure that it hasn't already been resolved.
         Sharing a link to a GitHub repository or preview deployment are best. Include any instructions to run the project in the README or the summary field below.
 
-        Don't have an available repo/preview to share? You can create online with our [StackBlitz template](https://stackblitz.com/edit/web3modal-next-template) quickly.
+        Don't have an available repo/preview to share? You can create online with our [StackBlitz template](https://stackblitz.com/edit/appkit-next-template) quickly.
```
Comment (ignaciosantise) [inline .github/ISSUE_TEMPLATE/bug_report.yml:14]: this url doesn't work

**Review by ignaciosantise — APPROVED** [review]

File: .changeset/config.json [hunk 5]
```diff
@@ -1,10 +1,9 @@
 {
   "$schema": "https://unpkg.com/@changesets/config@3.0.0/schema.json",
-  "changelog": ["@changesets/changelog-github", { "repo": "WalletConnect/web3modal" }],
+  "changelog": ["@changesets/changelog-github", { "repo": "reown-com/appkit" }],
   "commit": false,
   "fixed": [],
   "linked": [],
-  "access": "public",
```
Comment (ignaciosantise) [inline .changeset/config.json:7]: 👀

**Review by magiziz — APPROVED** [review]
Looks good to me. Just a few small suggestions.

File: apps/laboratory/src/pages/library/ethers5-no-email.tsx [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+  networks: [arbitrum, mainnet, optimism, polygon, zkSync, sepolia],
+  defaultNetwork: mainnet,
+  projectId: ConstantsUtil.ProjectId,
+  features: {
+    analytics: true,
+    email: false,
+    socials: []
+  },
+  termsConditionsUrl: 'https://walletconnect.com/terms',
+  privacyPolicyUrl: 'https://walletconnect.com/privacy',
```
Comment (ignaciosantise) [inline apps/laboratory/src/pages/library/ethers5-no-email.tsx:23]: remember to update this ones

**Review by lukaisailovic — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (tomiir) [reply .github/ISSUE_TEMPLATE/config.yml:4]: @lukaisailovic can we replace web3modal category with appkit on the monorepo issues ? 🙏

Comment (lukaisailovic) [reply .github/ISSUE_TEMPLATE/config.yml:4]: Yes. Lets do it.

**Review by svenvoskamp — APPROVED** [review]

**Review by enesozturk — APPROVED** [review]

**Pull request merged**